enhancements/network/on-prem-configurable-lb-dns-stack.md

```diff
@@ -0,0 +1,332 @@
+---
+title: on-prem-configurable-lb-dns-stack
+authors:
+  - "@yboaron"
+reviewers:
+  - "@sttts"
+  - "@deads2k"
+  - "@celebdor"
+  - "@bcrochet"
+  - "@bnemec"
+approvers:
+  - "@"
+creation-date: 2020-02-11
+last-updated: 2020-02-11
+status: implementable
+---
+
+# On-prem: configurable DNS/LB self-hosted stack
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+In the current implementation,  the self-hosted DNS/LB stack runs under the MCO umbrella by default in all on-prem platforms and there's no option to configure it.
+Some customers with high production needs usually have their own external Load Balancing and DNS resolution.
```

Comment:
what does "high production" means?

Reply:
I assume its customers that need/want better HA and resiliency.
Maybe @williamcaban  can elaborate on customers requirement for external LB for the API.

The two limitations with current  self-hosted LB architecture I'm aware of:
- all traffic goes through a single node (the one holds the VIP) 
- Node reboot/shutdown may break connections

Reply:
I read "high production needs" as production environments with a lot of traffic, but agree the phrasing is a little confusing. Maybe we can come up a way to explain it in terms of how the current solution is insufficient.

Reply:
OK, I'll update that.
10x